sdks/java/container/boot.go

```diff
@@ -104,6 +104,9 @@ func main() {
 		filepath.Join(jarsDir, "beam-sdks-java-harness.jar"),
 	}
 	for _, md := range artifacts {
+		if strings.HasPrefix(md.Name, "beam-runners-google-cloud-dataflow-java-fn-api-worker") {
```

Comment:
We should only make this check if the experiment is set. Also, the name will change to "dataflow-worker.jar" when the artifact bug is fixed.

Reply:
The purpose here is, if there is java worker jar in artifacts, then this jar should not be included into sdk harness classpath, which seems like we don't need to check experiment. wdyt?